src/bev_node/bev_node_main.cc

```diff
@@ -76,7 +76,7 @@ Eigen::Matrix3f ReadIntrinsicMatrix() {
   }
 }
 
-void InputImageCallback(const sensor_msgs::CompressedImageConstPtr msg) {
+void InputImageCallback(const sensor_msgs::ImageConstPtr& msg) {
```

Comment:
`image_transport` uses the same function call for both `raw` and `compressed` modes, automatically decompressing a compressed image before the function call. 

It's interesting that the required function signature takes a reference to a smart pointer rather than the smart pointer itself.